go.mod

```diff
@@ -18,6 +18,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3 //v1.11 will upgrade k8s APIs to v0.23.x
 )
 
+require github.com/aws/aws-sdk-go v1.44.68
```

Comment:
Can this be moved to the larger ( ) section above?

Reply:
Yes, sure. It was placed there after running `go mod tidy`.

Reply:
I've made the changes

Reply:
@bcle The changes have been pushed. You may take a look.